.snippets/code/node-operators/network-node/rpc-docker/docker-command.md

```diff
@@ -1,17 +1,16 @@
 --chain=/chain-network/container-YOUR_NETWORK_ID-raw-specs.json \
 --rpc-port=9944 \
 --name=para \
+--state-pruning=archive \
+--blocks-pruning=archive \
+--database=paritydb \
+--unsafe-rpc-external \
```

Comment:
Normally we don't recommend `--unsafe-rpc-external` or is this a flag that OpsLayer confirmed can be used? I think this is not a great security practice, so if we were to add this it should be under a note or a warning.